include/class.mailparse.php

```diff
@@ -664,7 +664,7 @@ function parse($stream) {
 
                     $data['thread_entry_recipients'][$source][] = sprintf('%s <%s>', $name, $email);
                 } elseif ($emailId) {
-                    $data['system_emails'][] = $emailId;
+                    $data['system_emails']['emails'][] = $emailId;
```

Comment:
I don't understand why you introduced `emails` key here  - it will absolutely break use of `system_emails` systemwide.